open_gpt/serve/flow.py

```diff
@@ -20,9 +20,9 @@ async def deploy(flow: str):
     import tempfile
 
     if os.path.isfile(flow):
-        return await CloudFlow(path=flow).deploy()
+        return await CloudFlow(path=flow)._deploy()
```

Comment:
nope, `_deploy` does not hold until the flow is ready. In our case, the `deploy()` provided from jcloud will display the progress logs in console which is exactly a good UX.